### ISSUE TITLE
Integrate uniform versioning guidelines

### ISSUE THREAD

**Issue by nickwalkmsft** [Microsoft/api-guidelines#554]
This is a suggestion to integrate the [uniform versioning doc](https://aka.ms/azsdk/uniform-versioning) and accompanying glossary from Azure/azure-rest-api-specs into the [Azure REST API guidelines doc](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#api-versioning), or at least reference it. It's a significant addition to the versioning guidelines already there.

**Comment by JeffreyRichter** [comment]
Yes, I agree that we should either link or integrate. I'd like to get more Azure service team feedback on the unified versioning doc before we formalize it more by linking/integrating.
